src/docs/binding-properties.md

```diff
@@ -53,21 +53,21 @@ public struct FrozenPoint {
 
     private unsafe void Set_x(nint value)
     {
-        IntPtr payload;
+        void* payload;
         try 
         {
             var metadata = ...;
             payload = (SwiftHandle)NativeMemory.Alloc(returnMetadata.Size);
```

Comment:
Is this still required? Sorry, not sure if you plan more changes :)

Reply:
Removed